packages/commonwealth/client/scripts/controllers/app/webWallets/keplr_ethereum_web_wallet.ts

```diff
@@ -49,34 +50,36 @@ class EVMKeplrWebWalletController implements IWebWallet<AccountData> {
     return this._offlineSigner;
   }
 
-  public async signLoginToken(
-    message: string,
-    address: string
-  ): Promise<string> {
+  getChainId() {
+    return this._chainId;
+  }
+
+  public async getRecentBlock(chainIdentifier: string) {
+    const url = `${window.location.origin}/cosmosAPI/${
```

Comment:
Use injected provider here -- evmos only, for now.

Reply:
AFAIK Keplr differs from Ethereum in that you just use `window.keplr` for sign things, it does not connect to the RPC by itself (unless you use `SigningStargateClient`). We just want to get block information from the RPC so it is enough to make a `StargateClient`.

Reply:
See https://github.com/hicommonwealth/commonwealth/pull/2373#discussion_r1059036742